.circleci/config.yml

```diff
@@ -68,24 +68,33 @@ workflows:
           <<: *only_development
 
       # pre-staging
-      - hokusai/test:
-          name: test
+      - yarn/run:
+          name: test-jest-v1
           <<: *not_staging_or_release
-          post-steps:
-            - run: mkdir -p ./coverage
-            - run:
-                name: Copy coverage artifacts
-                command: docker cp hokusai_metaphysics_1:/app/coverage ./
-                when: always
-            - codecov/upload:
-                file: ./coverage/lcov.info
+          script: "test:jest:v1"
+
+      - yarn/run:
+          name: test-jest-v2
+          <<: *not_staging_or_release
+          script: "test:jest:v2"
+
+          # TODO: Reenable code-cov uploads
+          # post-steps:
+          #   - run: mkdir -p ./coverage
+          #   - run:
+          #       name: Copy coverage artifacts
+          #       command: docker cp hokusai_metaphysics_1:/app/coverage ./
+          #       when: always
+          #   - codecov/upload:
+          #       file: ./coverage/lcov.info
```

Comment:
Have to follow up on this, though TBH I'm not sure how much value code coverage really brings to a web service. It makes a lot of sense for 3rd party libs as a means of building trust, but internal to a company I suspect codecov is largely ignored.

Reply:
You may have a point there, but we should probably have that discussion outside the scope of this change. Was it failing due to an overall lower coverage (given that the v1 tests are split out)?

Reply:
Its not that it was failing, its that I wasn't sure how to hop from the previous hokusai/test docker test context into a raw node context, eg this command up above: 
```
command: docker cp hokusai_metaphysics_1:/app/coverage ./
```

Reply:
Ahh, I see. You don't actually need that line. Jest has the coverage plugin so it'll automatically output the results to the `coverage` directory. All that was doing was copying it out of the container so that it could be uploaded. I think if you just deleted that run step it'd complete successfully.